lib/engine/game/g_1854/step/merge_minors.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1854
+      module Step
+        class MergeMinors < Engine::Step::Base
+          def actions(entity)
+            return [] if entity.corporation? && !entity.minor?
+            return [] if !entity.minor? || entity != current_entity #  || @game.done_this_round[entity]
+            return [] unless @game.mergeable?(entity)
+            return [] if target_corporations.empty?
+            return [] if @merge_target
```

Comment:
[Sorry, far too many suggestions close together, let's do this all at once].
```suggestion
            return [] unless entity == current_entity
            return [] unless entity.minor?
```
You don't need several of these checks.

- The `corporation?` check is redundant as you're checking `minor?`.
- AFAICT, `mergeable?` will always return true for a minor.
- `target_corporations.empty?` will only be true once there are no minors left to merge.
- `@merge_target` is for where you can merge more than two entities together.